tracing/src/span.rs

```diff
@@ -720,10 +720,41 @@ impl Span {
     ///
     /// If this span is disabled, or the resulting follows-from relationship
     /// would be invalid, this function will do nothing.
-    pub fn follows_from(&self, from: impl for<'a> Into<Option<&'a Id>>) -> &Self {
+    ///
+    /// # Example
+    ///
+    /// Setting a `follows_from` relationship through an `Option<Id>`:
+    /// ```
+    /// # use tracing::{span, Id, Level, Span};
+    /// # fn main() {
+    /// let span = span!(Level::INFO, "hello!");
+    /// let curr = Span::current();
```

Comment:
Note also that it's not really _necessary_ to use `Span::current()`; we might want to consider an example that shows something like

```rust
let span1 = span!(Level::INFO, "span_1");
let span2 = span!(Level::DEBUG, "span_2");
span2.follows_from(span1);

Reply:
Added.